pkg/registry/apis/dashboard/register.go

```diff
@@ -172,13 +174,18 @@ func (b *DashboardsAPIBuilder) GetAPIGroupInfo(
 		builder: b,
 	}
 
+	mode := grafanarest.Mode1
```

Comment:
We have to clarify the scope of our work, in which mode(s) do we need to enforce RBAC?